### PR TITLE
flip endianess if the system is little endian for signed shorts

### PR DESCRIPTION
**Pull request by Crim**
Probably should write a test or two to cover this before it gets merged in.

### CODE REVIEW

**Comment by Crim** [comment]
Aside from the coding style fails, I'm not so convinced I did this correctly yet.  I'd like to spend a bit more time playing with this and write up a few tests to validate I'm flipping the value properly.

**Comment by AlexKR** [comment]
Thank you, it works fine on my system.

File: src/Kafka/Protocol/Protocol.php [hunk 1]
```diff
@@ -230,5 +252,52 @@ protected static function checkLen($type, $bytes)
...
     // }}}
+    // {{{ public static function isSystemLittleEndian()
+
+    /**
+     * Determines if the computer currently running this code is big endian or little endian.
+     *
+     * @access public
+     * @return bool - false if big endian, true if little endian
+     */
+    public static function isSystemLittleEndian() {
```
Comment (AlexKR) [inline src/Kafka/Protocol/Protocol.php]: Put brace on next line to fix CI code-standart checking

**Comment by Crim** [comment]
Ok, I believe I worked this out, and wrote some tests as well.  @AlexKR If you could run the test suite on your machine since it's big endian and let me know if it still passes that'd be awesome.

My machine is little endian, and the tests all pass :)

**Comment by Crim** [comment]
@nmred it possible to get this bug fix merged in?  This has been creating issues in our production environment, and surely affecting others running 64bit systems.

Thanks!

**Comment by nmred** [comment]
Thanks @Crim

**Pull request merged**